src/Equinor.ProCoSys.IPO.Infrastructure/EntityConfigurations/CertificateConfiguration.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Equinor.ProCoSys.IPO.Domain.AggregateModels.InvitationAggregate;
+using Equinor.ProCoSys.IPO.Domain.AggregateModels.ProjectAggregate;
+using Equinor.ProCoSys.IPO.Infrastructure.EntityConfigurations.Extensions;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace Equinor.ProCoSys.IPO.Infrastructure.EntityConfigurations
+{
+    internal class CertificateConfiguration : IEntityTypeConfiguration<Certificate>
+    {
+        public void Configure(EntityTypeBuilder<Certificate> builder)
+        {
+            builder.ConfigurePlant();
+            builder.ConfigureCreationAudit();
+            builder.ConfigureConcurrencyToken();
+
+            builder.HasOne<Project>()
+                .WithMany()
+                .HasForeignKey(x => x.ProjectId)
+                .IsRequired()
+                .OnDelete(DeleteBehavior.NoAction);
+
+            builder.Property(x => x.PcsGuid)
+                .IsRequired();
+
+            builder
```

Comment:
Could not remember which one, but was it not so that Certificate could have only on McPck or CommPck? Should it not be HasOne on one of them?

Reply:
Will leave as many-to-many so that this is generic for certificates. Will also avoid potential problems with voiding of certificates if we have a many-to-one where we in theory could (if only looking at RFOC to mcpkg)